src/preprocess/index.ts

```diff
@@ -253,12 +253,14 @@ export type ImportLoader = (location: ImportLocation, baseLocation: ImportLocati
 export const importLoaders: {[key in ImportType]: ImportLoader} = {
 
   file: async (location, baseLocation) => {
-    if (_.some(_.filter(importTypes, 'file')), (typ: string) => baseLocation.startsWith(`${typ}:`)) {
-      logger.debug(`non file: import on baseLocation=${baseLocation} for location=${location}`)
-    }
     const resolvedLocation = pathmod.resolve(pathmod.dirname(baseLocation.replace('file:', '')), location.replace('file:', ''));
-    const data = (await bluebird.promisify(fs.readFile)(resolvedLocation)).toString();
-    return {resolvedLocation, data, doc: resolveDocFromImportData(data, resolvedLocation)}
+    try {
+      const data = (await bluebird.promisify(fs.readFile)(resolvedLocation)).toString();
+      return {resolvedLocation, data, doc: resolveDocFromImportData(data, resolvedLocation)}
+    } catch (e) {
+      throw new Error(
+        `"${baseLocation}" has a bad import "$imports: ... ${location}". ${e}`);
```

Comment:
Not super familiar with the iidy codebase, but what are `baseLocation` and `location`? I.e. what's an example of an error message that a user would see?

Reply:
`baseLocation` is the file that contains the `$imports` entry. `location` is the value of the import.
`error: "/Users/tavis/src/ub/cf_templates/examples/bad-imports.yaml" has a bad import "$imports: ... blah". Error: ENOENT: no such file or directory, open '/Users/tavis/src/ub/cf_templates/examples/blah'`